deployment/permissions/service.entryservernode.ini

```diff
@@ -0,0 +1,26 @@
+[action_permission_items]
+permissionItem1.service = entryservernode
```

Comment:
Based on your decision about keeping/removing the add/update/etc ... update this file accordingly.